handlers/new.go

```diff
@@ -78,5 +87,7 @@ func NewTokenHandler(w http.ResponseWriter, r *http.Request) {
 
 	log.Println("New cluster created", token)
 
-	fmt.Fprintf(w, "https://discovery.etcd.io/"+token)
+	domain := getDomain()
+
+	fmt.Fprintf(w, domain+"/"+token)
```

Comment:
Feel free to pick this up: https://github.com/TimeIncUK/discovery.etcd.io/compare/configurable#diff-2ee42baea70799acc66e2b0b21a772f2R71
I can send it either together or separately as a PR if you wish.